### ISSUE TITLE
It would be better to remove the actions 'Clone.. & Copy Blob Container' for one blob container if the setting 'Disable Usage of Keys' is checked

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#5935]
**Storage Explorer Version**: 1.25.0-dev
**Build Number**: 20220715.7
**Branch**: main
**Platform/OS**: Windows 10/Linux Ubuntu 20.04/MacOS Monterey 12.4 (Apple M1 Pro)
**Architecture**: ia32/x64
**How Found**: From running test cases
**Regression From**: Previous release (1.24.3)

## Steps to Reproduce ##
1. Launch Storage Explorer -> Go to 'Settings -> Services' -> Check 'Disable Usage of Keys'.
2. Restart Storage Explorer.
3. Expand one storage account -> Blob Containers.
4. Right click one blob container -> Observe the actions on the context menu.
5. Check there is no actions 'Clone... & Copy Blob Container'.

## Expected Experience ##
It would be better to remove the actions 'Clone... & Copy Blob Container'.
![image](https://user-images.githubusercontent.com/41351993/179182069-9bdeb791-893c-47b4-aa26-fa0fb9814ae1.png)

## Actual Experience ##
There are actions 'Clone... & Copy Blob Container'.
![image](https://user-images.githubusercontent.com/41351993/179182222-493a5c49-f0a2-4e97-94da-58c4060e40b9.png)

## Additional Context ##
It would be better to move 'Clone...' action for one table if the setting 'Disable Usage of Keys' is checked.
![image](https://user-images.githubusercontent.com/41351993/179182604-3293d782-284d-4d0c-b2c5-ca7460947190.png)

**Comment by craxal** [comment]
Yes, Clone works for tables. Therefore, we should not be removing Clone or Copy menu items.
